app.vue

```diff
@@ -10,7 +10,7 @@ useServerSeoMeta({
 
 <template>
   <Html>
-    <Body class="bg-neutral-100 dark:bg-neutral-900 dark:text-white transition-all duration-300">
```

Comment:
the reason we added `transition` here is for switching between dark mode and light mode

Reply:
Ah, thanks for the context. I've reduced the transition to `transition-colors` which should satisfy both issues.